pkg/binding/bindingpolicy.go

```diff
@@ -279,10 +279,11 @@ type mrObject interface {
 // testObject tests if the object matches the given tests.
 // The returned tuple is:
 //   - bool: whether the object matches ANY of the tests
+//   - bool: whether any of the tests says CreateOnly==true
 //   - sets.Set[string]: the UNION of the statuscollector names that appear within
 //     EACH of the tests that the object matches
```

Comment:
This one already has "that the object matches".

Reply:
Yes. I meant to unify the syntax by making it:

```
//   - sets.Set[string]: the UNION of the statuscollector names that appear within
//     EACH of the matching tests
```

Reply:
On second thought, I think "matching test" is a bit ambiguous because the purpose of a test is matching, so the "matching" in that phrase _could_ be read as a description of simply the test.

I have pushed a revision that unifies the wording, going with wording that is explicit about the test matching the object.